data/sql/updates/pending_db_world/rev_1632156508775520500.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1632156508775520500');
+
+UPDATE `creature_template` SET `maxlevel` = 21 WHERE `entry` = 1039 AND `name` LIKE '%Fen Dweller%';
```

Comment:
`AND `name` LIKE '%Fen Dweller%'`
I don't think this is necessary, because the entry is already unique

Reply:
true, fixed.